### PR TITLE
Changes species' `inherent_traits` to alphabetical order

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
Ye that.

## Why It's Good For The Game
Every now and then I go read species' code to answer someone's question about one of the 26 kinds of golem and need to strain my eyes looking at those oddly ordered lists.

Yadda yadda readability or something.

## Changelog
Not player facing

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm [hunk 1]
```diff
@@ -137,7 +137,6 @@ Lizard subspecies: ASHWALKERS
 		TRAIT_CAN_STRIP,
 		TRAIT_CHUNKYFINGERS,
 		TRAIT_VIRUSIMMUNE,
-		//TRAIT_LITERATE, ashwalkers are uneducated hillbillies from lavaland
```
Comment (timothymtorres) [inline code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:140]: This is a code comment to show that the decision to not make them literate was deliberate.

File: code/modules/mob/living/carbon/human/species_types/monkeys.dm [hunk 2]
```diff
@@ -25,11 +25,10 @@
 	)
 	inherent_traits = list(
 		TRAIT_CAN_STRIP,
-		TRAIT_VENTCRAWLER_NUDE,
+		TRAIT_GUN_NATURAL,
 		TRAIT_PRIMITIVE,
+		TRAIT_VENTCRAWLER_NUDE,
 		TRAIT_WEAK_SOUL,
-		TRAIT_GUN_NATURAL,
-		//TRAIT_LITERATE, monkeys shouldn't be able to read or write
```
Comment (timothymtorres) [inline code/modules/mob/living/carbon/human/species_types/monkeys.dm:32]: This is a code comment to show that the decision to not make them literate was deliberate.

File: code/modules/mob/living/carbon/human/species_types/lizardpeople.dm (hunk 1 above)
Comment (GuillaumePrata) [reply code/modules/mob/living/carbon/human/species_types/lizardpeople.dm:140]: Reverted my change then.

File: code/modules/mob/living/carbon/human/species_types/monkeys.dm (hunk 2 above)
Comment (GuillaumePrata) [reply code/modules/mob/living/carbon/human/species_types/monkeys.dm:32]: Reverted my change then.

**Pull request merged**